strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -332,4 +345,42 @@ public ResponseEntity removeRepository(@RepositoryMapping Repository repository,
         }
     }
 
+    @ApiOperation(value = "Deletes recursively the content of path located under the defined storageId and repositoryId.")
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "The path was deleted successfully."),
+                            @ApiResponse(code = 404, message = "The path cannot be deleted because storageId,repositoryId,or path was not found!"),
+                            @ApiResponse(code = 500, message = "Failed to remove the path") })
+    @DeleteMapping(value = "/delete", consumes = MediaType.APPLICATION_JSON_VALUE, produces = { MediaType.TEXT_PLAIN_VALUE,
+                                                                                                MediaType.APPLICATION_JSON_VALUE })
+    public ResponseEntity removePath(@ApiParam(value = "The path object", required = true)
+                                     @RequestBody @Validated({ Default.class,
+                                                               ProxyConfigurationFormChecks.class})
+                                                 PathForm pathForm,
+                                     BindingResult bindingResult,
+                                     @RequestHeader(HttpHeaders.ACCEPT) String accept)
+    {
+
+        if(bindingResult.hasErrors())
+        {
+            return getFailedResponseEntity(HttpStatus.NOT_FOUND, FAILED_PATH_REMOVAL_FORM_ERROR, accept);
+        }
+
+        try
+        {
+            final RepositoryPath repositoryPath = repositoryPathResolver.resolve(pathForm.getStorageId(),
+                    pathForm.getRepositoryId(),
+                    pathForm.getPath());
+            
+            artifactManagementService.delete(repositoryPath, pathForm.isForce());
+            logger.info("Deleting Path {}:{}/{}...", pathForm.getStorageId(), pathForm.getRepositoryId(), pathForm.getPath());
```

Comment:
Cosmetic: Could you keep `logger` blocks separated by a line before and after? This makes it a bit more readable. Thanks! :)